src/policy/policy.cpp

```diff
@@ -179,6 +181,13 @@ bool IsStandardTx(const CTransaction& tx, std::string& reason)
             return false;
         }
 
+        if (txout->IsZerocoinMint())
+            nMintCount++;
+        if (nMintCount >= Params().Zerocoin_PreferredMintsPerTransaction()) {
+            reason = "zcmint_spam";
+            return false;
+        }
```

Comment:
If we are changing IsStandardTx, this would fork the chain unless it is wrapped in the BIP? Or am I mistaken?